brainreg_napari/register.py

```diff
@@ -409,14 +448,77 @@ def run(
             smoothing_sigma_floating,
             histogram_n_bins_floating,
             histogram_n_bins_reference,
+            save_original_orientation,
+            brain_geometry,
         )
         worker.returned.connect(add_image_layers)
         worker.start()
 
     @widget.reset_button.changed.connect
     def restore_defaults(event=None):
         for name, value in DEFAULT_PARAMETERS.items():
-            if name != "atlas_key":
+            if name not in ["atlas_key", "brain_geometry"]:
                 getattr(widget, name).value = value
 
+    @widget.check_orientation_button.changed.connect
+    def check_orientation(event=None):
+        """
+        Function used to check that the input orientation is correct. To do so it transforms the input data
+        into the requested atlas orientation, compute the average projection and displays it alongside the atlas. It
+        is then super easy for the super to identify which dimension should be swapped and avoid running the pipeline
```

Comment:
`super easy for the super`? Not sure I follow

Reply:
For the user not super, I'll modify it.